lib/scripts/windows/run-datadog.bat

```diff
@@ -0,0 +1,2 @@
+echo "Starting Datadog Agent"
+powershell "C:\Users\vcap\app\datadog\scripts\run-datadog.ps1"
```

Comment:
Should this be configurable?

Reply:
I don't think so? It just runs the powershell script we provide